src/UseCaseContext.ts

```diff
@@ -16,18 +16,16 @@ export default class UseCaseContext {
     dispatcher: UseCase;
 
     /**
-     * @param   [dispatcher]    dispatcher is Dispatcher or parent UseCase.
-     * @public
+     * @param   dispatcher
+     *  The parent UseCase.
      */
     constructor(dispatcher: UseCase) {
```

Comment:
> Insteadof this, We can add null default value to UseCaseExecutor constructor.

`UseCaseExecutor.parent` has already been `UseCase | null`. And there is no error at the HEAD of this branch. Really do we need change to what you suggested?